packages/conventional-commits-parser/src/cli/.eslintrc.json

```diff
@@ -0,0 +1,5 @@
+{
+  "rules": {
+    "import/unambiguous": "off"
+  }
+}
```

Comment:
you should define overrides instead in main eslint config

Reply:
https://github.com/conventional-changelog/conventional-changelog/pull/1126/files#diff-6884918dc8291219be508e05e28965b958c734def85324f3b53858ea4702090fR2

https://eslint.org/docs/latest/use/configure/configuration-files#using-configuration-files

there is `"root": true` property in root eslint config, so all others configs are extending parent config